cmd/global-heal.go

```diff
@@ -297,24 +303,27 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []BucketIn
 		})
 
 		if err != nil {
+			// Set this such that when we return this function
+			// we let the caller retry this disk again for the
+			// buckets it failed to list.
+			retErr = err
 			logger.LogIf(ctx, err)
-			return err
+			continue
```

Comment:
This is the only functional change, correct?

The only difference I see is that it will skip to the next bucket.

Reply:
yes @klauspost its also a way to not prematurely fail and proceed to heal as much as we can.